ndk-build/src/ndk.rs

```diff
@@ -17,7 +17,7 @@ impl Ndk {
             let mut sdk_path = std::env::var("ANDROID_HOME").ok();
             if sdk_path.is_some() {
                 println!(
-                    "Warning: You use environment variable ANDROID_HOME that is deprecated.\
+                    "Warning: You use environment variable ANDROID_HOME that is deprecated. \
                  Please, remove it and use ANDROID_SDK_ROOT instead. Now ANDROID_HOME is used"
```

Comment:
I'm not a fan of this warning. Any objections to flipping it around and using non-deprecated `ANDROID_SDK_ROOT` (just like `ANDROID_NDK_ROOT` below) before resorting to legacy `ANDROID_HOME`?

We might still print a warning if legacy variables are set, and mention they are not used.

Besides, turns out `ANDROID_NDK_HOME` and `ANDROID_NDK_PATH` are [a thing](https://github.com/actions/virtual-environments/blob/fa9109cc62be11105688bc7aa2305864a9dc762c/images/win/scripts/Installers/Update-AndroidSDK.ps1#L82-L83) as well, though I've never seen `NDK_HOME`.

Reply:
I think it's derived from here https://developer.android.com/studio/command-line/variables#envar

Reply:
Right, despite its deprecation it still has precedence over `ANDROID_SDK_ROOT`. If we want to adhere to these rules perhaps we should implement number 3 as well:

> If `ANDROID_HOME` is defined but does not exist or does not contain a valid SDK installation, the value in `ANDROID_SDK_ROOT` is used instead.

Reply:
Even though I'm also more in favor of dropping support for `ANDROID_HOME` or reordering priorities with `ANDROID_SDK_ROOT` being higher on the list